src/scvi/module/_vae.py

```diff
@@ -360,7 +370,7 @@ def _regular_inference(
         if self.log_variational:
             x_ = torch.log1p(x_)
 
-        if cont_covs is not None and self.encode_covariates:
+        if cont_covs is not None:
             encoder_input = torch.cat((x_, cont_covs), dim=-1)
```

Comment:
Why do you always concatenate continous cov to expression instead of handling them separately as a covariate (sam as for one-hot)?